### PR TITLE
[#4115] Govmetric changes

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes open-formulieren/open-forms#4115

:sparkles: Needs forward-port :sparkles: 

To do:
- Discuss the modal problem => Discussed: as long as the GovMetric questionnaire opened takes focus, then it's okay.

I could not get the questionnaire to open in an iframe :disappointed: 

Screenshot from 2024-04-12 16-58-56

### CODE REVIEW

File: src/components/analytics/GovMetricSnippet.js [hunk 1]
```diff
@@ -1,42 +1,102 @@
...
+            className="gm_sidebar_anchor"
+            title={intl.formatMessage({
+              description: 'Feedback face title - Good',
+              defaultMessage: "Rate this form as 'Good'",
+            })}
+            target="_blank"
+            rel="nofollow noreferrer"
+          >
+            <img
+              src="https://websurveys2.govmetric.com/imgs/smileys/medium/Good.png"
```
[the remark this answers is not in the document]
Comment (SilviaAmAm) [reply src/components/analytics/GovMetricSnippet.js:44]: The large ones are huuuge :sweat_smile: 
Screenshot from 2024-04-19 13-18-35

[the remark this answers is not in the document]
Comment (Viicos) [reply src/components/analytics/GovMetricSnippet.js:44]: Hum but we could set a size attribute on our `img` elements right?

[the remark this answers is not in the document]
Comment (SilviaAmAm) [reply src/components/analytics/GovMetricSnippet.js:44]: We could! But I think that the small images are ok?

[the remark this answers is not in the document]
Comment (Viicos) [reply src/components/analytics/GovMetricSnippet.js:44]: I found them a bit pixelated but this isn't that much of an issue, I think we can discuss this during refinement + regarding embedding them locally

[the remark this answers is not in the document]
Comment (SilviaAmAm) [reply src/components/analytics/GovMetricSnippet.js:44]: I added a new issue about the faces being blocked: https://github.com/open-formulieren/open-forms/issues/4208

**Pull request merged**